src/odemis/gui/model/__init__.py

```diff
@@ -1058,6 +1062,14 @@ def __init__(self, main):
                 "mirror_y": (10e-6, [100e-9, 1e-3], "mirror", {"y"}),
             })
 
+        if main.spec_switch:
+            # SPARCv2 light aligner dichroic mirror and spec switch foldable mirror
+            ss_def.update({
+                "light_aligner": (1e-6, [100e-9, 1e-4], "light_aligner", None),
```

Comment:
No need to place this in a `if` block. You can directly add the axes to ss_def. If the axes are not present, the config will not be used.